go/client/commands_common.go

```diff
@@ -34,6 +34,7 @@ func GetCommands(cl *libcmdline.CommandLine, g *libkb.GlobalContext) []cli.Comma
 		NewCmdDumpPushNotifications(cl, g),
 		NewCmdEncrypt(cl, g),
 		NewCmdGit(cl, g),
+		NewCmdHome(cl, g),
```

Comment:
Should this be a dev only command, at least for now?

Reply:
how about a hidden command?

Reply:
yeah, that's sounds fine